docs/usage_guide.rst

```diff
@@ -847,7 +847,7 @@ An xpub does not allow spending your coins but provides information about your w
 ENS names resolution
 =====================
 
-rotki automatically resolves ens name for each of your ethereum accounts. If there is a primary ens name specified for an account, this name will be shown instead of raw blockchain address across the app. You can find the blockchain address by hovering the mouse over the ens name.
+rotki automatically resolves ens name for each of your evm accounts. If there is a primary ens name specified for an account, this name will be shown instead of raw blockchain address across the app. You can find the blockchain address by hovering the mouse over the ens name.
```

Comment:
```suggestion
rotki automatically resolves ens name for each of your evm accounts. If there is a primary ens name specified for an account, this name will be shown instead of a raw blockchain address across the app. You can find the blockchain address by hovering the mouse over the ens name.
```